Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM alpine:3.4
+FROM node:8.1-alpine
```

Comment:
And while I'm at it I'll update to the Node version, I do want Node 8 and NPM 5.